### DISCUSSION TITLE
How to create a new user or login in local? It's always showing server is currently under maintainence.

### DISCUSSION THREAD

**Discussion by nishu-murmu** [monkeytypegame/monkeytype#3477] [Questions] [answered]
![Screenshot 2022-08-30 at 2 59 16 PM](https://user-images.githubusercontent.com/71123337/187402005-18487c8d-b8f2-4877-a53d-f0da3a0cccc0.png)

**(Edit)**
![Screenshot 2022-08-30 at 3 05 52 PM](https://user-images.githubusercontent.com/71123337/187403503-99f6e120-2cba-4f8c-b19b-f70c550e8677.png)
no anitcheat module found, What does that means?

**Comment by Miodec** [comment, marked as answer]
Make sure maintenance mode is disabled in the server config (there is a button on the left side of the frontend that will take you to a panel where you can edit the server configuration)
